detectable/src/test/java/com/synopsys/integration/detectable/detectables/pip/parser/RequirementsFileDetectableUtils.java

```diff
@@ -0,0 +1,16 @@
+package com.synopsys.integration.detectable.detectables.pip.parser;
+
+import java.io.File;
+
+import com.synopsys.integration.bdio.model.externalid.ExternalId;
+
+public class RequirementsFileDetectableUtils {
```

Comment:
I'm a bit confused on what this class does. Is it for a test? I didn't see it being referenced at first glance.

Reply:
Missed removing this one, thanks. Will clean it up - not used anywhere.